data/backend/reinit.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+rm -f db.sqlite3
+rm -f textile/migrations/*
+rm -f authentication/migrations/*
```

Comment:
I don't understand why we wouldn't want to keep the migrations versioned.

Reply:
They are meant to be versionned, as soon as @ccomb is happy with the "persistent db" setup, i think?

Reply:
Sure they will be versionned,  I didn't want to version many migrations during dev (even if django merges them efficiently)

Reply:
@ccomb are you aware of the `manage.py squashmigrations` command?